examples/toolkit_showcase/toolkit_showcase.ipynb

```diff
@@ -24,7 +24,7 @@
     {
```

Comment:
<p>(blocking) Could you clean up this output/rerun with a config that doesn't produce this warning? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>